backend/mail-service/src/jvmMain/resources/db/migration/V2__emailSettings.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE email_settings(
+    username varchar(255) UNIQUE NOT NULL,
```

Comment:
We don't have any hard guarantees that usernames don't exceed 255 characters.

```suggestion
    username TEXT PRIMARY KEY,
```